content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -53,11 +53,16 @@ Support for the Topology Manager requires `TopologyManager` [feature gate](/docs
 
 The Topology Manager currently:
 
- - Works on Nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/)
- - Works on Pods making CPU requests or Device requests via extended resources
+ - Aligns Pods of all QoS classes.
+ - Aligns the requested resources that Hint Provider provides topology hints for.
 
 If these conditions are met, Topology Manager will align the requested resources.
 
+Note: 
+
+ - To align CPU resources with another resources, the CPU Manager should be enabled and proper CPU Manager policy should be configured on a Node. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).

```

Comment:
Perhaps change to:
To align CPU resources of other resources,

Reply:
updated